<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gpiq_pdsxx</class>
 <widget class="QDialog" name="gpiq_pdsxx">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gpiq_pdsxx</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <widget class="QLabel" name="gpiq_pdsxx_xxprg_l">
     <property name="text">
      <string>=gpiq_pdsxx=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="gpiq_pdsxx_del_c">
     <property name="text">
      <string>udalit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="gpiq_pdsxx_xxsma_l">
       <property name="text">
        <string>=Галанин П. И.=</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QFrame" name="gpiq_pdsxx_b1">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="gpiq_pdsxx_dokk_l">
        <property name="text">
         <string>gpi_kod dokumenta</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpiq_pdsxx_dokk_e"/>
      </item>
      <item>
       <widget class="QLineEdit" name="gpiq_pdsxx_dokn_e"/>
      </item>
      <item>
       <widget class="QLabel" name="gpiq_pdsxx_dokd_l">
        <property name="text">
         <string>ot</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="gpiq_pdsxx_dokd_e"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="gpiq_pdsxx_exit_c">
     <property name="text">
      <string>vixod</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="gpiq_pdsxx_xxtitle_l">
     <property name="text">
      <string>gpiq_vvod i raznoska pervisnix dokumentov</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="gpiq_pdsxx_grid_c">
       <property name="text">
        <string>vibrat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gpiq_pdsxx_back_c">
       <property name="text">
        <string>nazad</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gpiq_pdsxx_next_c">
       <property name="text">
        <string>vpered</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="gpiq_pdsxx_add_c">
       <property name="text">
        <string>dobavit</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QFrame" name="gpiq_pdsxx_b2">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="3" column="1">
       <widget class="QLineEdit" name="gpiq_pdsxx_to_e">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gpiq_pdsxx_rub_e"/>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="gpiq_pdsxx_to_c">
        <property name="text">
         <string>gpi_operacia</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>12</number>
        </property>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="gpiq_pdsxx_kr_l">
          <property name="text">
           <string>gpi_Kr</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gpiq_pdsxx_kr_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gpiq_pdsxx_krn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="gpiq_pdsxx_db_l">
          <property name="text">
           <string>gpi_Db</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gpiq_pdsxx_db_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gpiq_pdsxx_dbn_e">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="gpiq_pdsxx_rzd1_l">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="gpiq_pdsxx_pdtrj_c">
        <property name="text">
         <string>gpi_raznoska dokumenta (zanesenie provodok v reg jurnal)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="gpiq_pdsxx_rub_l">
        <property name="text">
         <string>gpi_summa</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>gpiq_pdsxx_dokk_e</tabstop>
  <tabstop>gpiq_pdsxx_dokn_e</tabstop>
  <tabstop>gpiq_pdsxx_dokd_e</tabstop>
  <tabstop>gpiq_pdsxx_rub_e</tabstop>
  <tabstop>gpiq_pdsxx_to_c</tabstop>
  <tabstop>gpiq_pdsxx_to_e</tabstop>
  <tabstop>gpiq_pdsxx_db_e</tabstop>
  <tabstop>gpiq_pdsxx_dbn_e</tabstop>
  <tabstop>gpiq_pdsxx_kr_e</tabstop>
  <tabstop>gpiq_pdsxx_krn_e</tabstop>
  <tabstop>gpiq_pdsxx_pdtrj_c</tabstop>
  <tabstop>gpiq_pdsxx_grid_c</tabstop>
  <tabstop>gpiq_pdsxx_back_c</tabstop>
  <tabstop>gpiq_pdsxx_next_c</tabstop>
  <tabstop>gpiq_pdsxx_add_c</tabstop>
  <tabstop>gpiq_pdsxx_del_c</tabstop>
  <tabstop>gpiq_pdsxx_exit_c</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
